docs/how-to-use-the-playground.md

```diff
@@ -121,3 +121,45 @@ FROM "metalake_demo.catalog_postgres".hr.employees AS e,
 WHERE e.employee_id = p.employee_id AND p.employee_id = s.employee_id
 GROUP BY e.employee_id,  given_name, family_name;
 ```
+
+### Lakehouse practice
+
+If you want to migrate your business from Hive to Iceberg. Some tables will use Hive, and the other tables will use Iceberg.
+Gravitino provides an Iceberg REST catalog service, too. You can will use Spark to access REST catalog to write the table data.
+Then, you can use Trino to read the data from the Hive table joining the Iceberg table.
+
+1. Login Spark container and execute the steps.
+
+```shell
+docker exec -it playground-spark bash
+```
+
+```shell
+spark@7a495f27b92e:/$ cd /opt/spark && /bin/bash bin/spark-sql 
```

Comment:
add some description about the spark sql client configured to use Iceberg rest catalog which is Gravitino Iceberg REST catalog service?

Reply:
It's hard to describe. Because we use REST catalog service in the Spark configuration. But for Gravitino we use Iceberg jdbc catalog. I can't describe too much in case of misunderstanding of newcomers.

Reply:
Fixed.